### ISSUE TITLE
pg:backups info command causes errors

### ISSUE THREAD

**Issue by JacobEvelyn** [heroku/heroku#1405]
When I run `heroku pg:backups info b001 -a my-app` or `heroku pg:backups info -a my-app` I get the following error:

```
Error submitting error.

 !    Heroku client internal error.
 !    Search for help at: https://help.heroku.com
 !    Or report a bug at: https://github.com/heroku/heroku/issues/new

    Error:       undefined method `-' for nil:NilClass (NoMethodError)
    Command:     heroku pg:backups info b001 -a panorama-rainbow
    Plugins:     heroku-pg-extras
                 heroku-push

    Version:     heroku-toolbelt/3.25.0 (x86_64-darwin10.8.0) ruby/1.9.3


    More information in /Users/jacobevelyn/.heroku/error.log
```

My local `.heroku/error.log` file looks like this:

```
exception class/object expected
/Users/jacobevelyn/.heroku/client/lib/heroku/rollbar.rb:9:in `raise'
/Users/jacobevelyn/.heroku/client/lib/heroku/rollbar.rb:9:in `error'
/Users/jacobevelyn/.heroku/client/lib/heroku/helpers.rb:441:in `styled_error'
/Users/jacobevelyn/.heroku/client/lib/heroku/cli.rb:52:in `rescue in start'
/Users/jacobevelyn/.heroku/client/lib/heroku/cli.rb:35:in `start'
/usr/bin/heroku:24:in `<main>'
Heroku client internal error.
undefined method `-' for nil:NilClass
/Users/jacobevelyn/.heroku/client/lib/heroku/command/pg_backups.rb:227:in `backup_status'
/Users/jacobevelyn/.heroku/client/lib/heroku/command/pg_backups.rb:53:in `backups'
/Users/jacobevelyn/.heroku/client/lib/heroku/command.rb:223:in `run'
/Users/jacobevelyn/.heroku/client/lib/heroku/cli.rb:40:in `start'
/usr/bin/heroku:24:in `<main>'
exception class/object expected
/Users/jacobevelyn/.heroku/client/lib/heroku/rollbar.rb:9:in `raise'
/Users/jacobevelyn/.heroku/client/lib/heroku/rollbar.rb:9:in `error'
/Users/jacobevelyn/.heroku/client/lib/heroku/helpers.rb:441:in `styled_error'
/Users/jacobevelyn/.heroku/client/lib/heroku/cli.rb:52:in `rescue in start'
/Users/jacobevelyn/.heroku/client/lib/heroku/cli.rb:35:in `start'
/usr/bin/heroku:24:in `<main>'
Heroku client internal error.
undefined method `-' for nil:NilClass
/Users/jacobevelyn/.heroku/client/lib/heroku/command/pg_backups.rb:227:in `backup_status'
/Users/jacobevelyn/.heroku/client/lib/heroku/command/pg_backups.rb:53:in `backups'
/Users/jacobevelyn/.heroku/client/lib/heroku/command.rb:223:in `run'
/Users/jacobevelyn/.heroku/client/lib/heroku/cli.rb:40:in `start'
/usr/bin/heroku:24:in `<main>'
...
```

Let me know if there's anything I can do to help fix this.

**Comment by dickeyxxx** [comment]
@uhoh-itsmaciek any ideas on this one? I put a fix in for the rollbar error (rollbar is rate limited, that's not the real error, but I fixed it in #1408)

the real issue is this:
```
undefined method `-' for nil:NilClass
/Users/jacobevelyn/.heroku/client/lib/heroku/command/pg_backups.rb:227:in `backup_status'
/Users/jacobevelyn/.heroku/client/lib/heroku/command/pg_backups.rb:53:in `backups'
/Users/jacobevelyn/.heroku/client/lib/heroku/command.rb:223:in `run'
/Users/jacobevelyn/.heroku/client/lib/heroku/cli.rb:40:in `start'
/usr/bin/heroku:24:in `<main>'
```

**Comment by uhoh-itsmaciek** [comment]
Hmm, I haven't seen this. Looking into it.

**Comment by dickeyxxx** [comment]
@JacobEvelyn could you try running `heroku plugins:update` to see if it fixes the problem? If that doesn't work, maybe try uninstalling that `heroku-pg-extras` and retrying to see if there is a conflict there?

**Comment by uhoh-itsmaciek** [comment]
Ok, sorry about the delay. I got sick and it took me a while to catch up on everything. I think I made some incorrect assumptions in the client code w.r.t. the data available about each backup: I had added some additional info, but older backups did not have it, and this step failed. Rather than requiring a client update, I just filled in the missing info for older backups, so this should be working correctly now. Let me know if you still have problems @JacobEvelyn.
